src/js/utils/dom.js

```diff
@@ -493,17 +506,6 @@ export function getPointerPosition(el, event) {
 }
 
 /**
- * Determines, via duck typing, whether or not a value is a DOM element.
```

Comment:
It moved up above because it was used before it was defined (which works because hoisting, but... it's against the rules).